nanopub/sign_utils.py

```diff
@@ -111,7 +111,7 @@ def publish_graph(g: ConjunctiveGraph, use_server: str = NANOPUB_SERVER_LIST[0])
     headers = {'Content-Type': 'application/trig'}
     # Used by nanopub-java: {'Content-Type': 'application/x-www-form-urlencoded'}
     data = g.serialize(format="trig")
-    r = requests.post(use_server, headers=headers, data=data)
+    r = requests.post(use_server, headers=headers, data=data.encode('utf-8'))
```

Comment:
Good question, I don't remember exactly why I put it here... Probably because I met issues with publishing graphs that contained non UTF-8 characters (or maybe it was added there trying to fix an unrelated issue and I forgot to remove it afterward)

But I am not even sure it is really needed tbh (this encoding is already handled by rdflib or requests). I removed it and all tests passes. Let's go without it, well spotted thanks :)